spec/features/comments_feature_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+feature 'Comments' do
+  # As a user
+  # So that I can share my thoughts on a picture
+  # I would like to write a comment on a picture
+
+  context 'when signed in' do
+  let!(:user){User.create(name: 'Betty Draper', email: 'betty@drapers.com',
+                          username: 'bettydraper', password: 'pigeons123',
+                          password_confirmation: 'pigeons123')}
+    scenario 'a user can post a comment' do
+      sign_in(user.email, user.password)
+      post_picture
+      click_link 'Add a comment'
+      fill_in 'Comment', with: 'A nice comment'
+      click_button 'Comment'
+      expect(page).to have_content 'A nice comment'
+    end
+  end
+
+  context 'when not signed in' do
+    let!(:user){User.create(name: 'Betty Draper', email: 'betty@drapers.com',
```

Comment:
Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)<br>Style/MultilineBlockLayout: Block body expression is on the same line as the block start.